packages/falso/src/lib/percentages.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  fake,
+  FakeOptions,
+  getRandomInRange,
+  RandomInRangeOptions,
+} from './core/core';
+
+export interface RandomNumberOptions extends RandomInRangeOptions, FakeOptions {
+  amount: number;
+  fraction?: number;
+}
+
+/**
+ * Generate a list of percentages that add up to 100%.
+ *
+ * @category general, math
+ *
+ * @example
+ *
+ * randPercentages() // default amount is 2 fraction is 3
+ *
+ * @example
+ *
+ * randPercentages({ amount: 3, length: 10 })
+ *
+ * @example
+ *
+ * randPercentages({ amount: 3 })
+ *
+ */
+export function randPercentages<Options extends RandomNumberOptions = never>(
```

Comment:
- I'm not too fond of the name. Moreover, it can be any number and not just "percentages".
- Can we make the return type more strict?

Reply:
Maybe it's better to call the function "randSummedNumbers" and add a "total value" to make the numbers sum up to that value. I'm not really sure what I would name this function though "randSummedNumbers" seems like a weird name too.

I found [this](https://stackoverflow.com/a/69248547/12371355) stack overflow answer, maybe it's better to return something like that

Reply:
Yes, we can use this type.

Regarding the name, maybe we can call it `randAggregation({ totalValue: 100 })`

Reply:
Yes, randAggregation is really good, I'll update this PR!